src/index.js

```diff
@@ -146,7 +147,9 @@ function App() {
           path="/parent/dashboard"
           exact
           component={() => (
-            <ParentDashboard LoadingComponent={ParentLoadingComponent} />
+            // <ParentDashboard LoadingComponent={ParentLoadingComponent} />
```

Comment:
make sure you leave a comment stating this is the old parent dashboard that can either be removed or used again in the future.

Reply:
I commented on that particular part with the old parent dashboard